examples/hello_server.cc

```diff
@@ -15,19 +15,24 @@ class HelloHandler : public Http::Handler {
     HTTP_PROTOTYPE(HelloHandler)
 
     void onRequest(const Http::Request& request, Http::ResponseWriter response) {
-        response.send(Http::Code::Ok, "Hello World");
+        (void) request;
```

Comment:
Did you want to change this to "UNUSED(request)" as well?

Reply:
Sure. I used the void cast here because it's an example and I thought the `UNUSED` macro might confuse new users.

Reply:
Well, the UNUSED macro is used throughout the rest of the code, and imho, it is kinda self-explanatory.  Please change this to use the macro.  My main goal here is to remove the C-style cast.